shopify-client/src/components/Header.vue

```diff
@@ -35,9 +39,15 @@
 <script>
 export default {
   name: "Header",
+  methods:{
+    submitSearch: function(){
+      if(this.keyword){
+        window.location.href = '/app/shops/List?keyword=' + this.keyword;
```

Comment:
so this seaches shops only?

Reply:
yes shops only
The keyword will be used in the backend to obtain all shops that has the keyword in shop name or shop description

Reply:
Ok, I thought we wanted the search bar to be able to search shops, products, tags, categories. I understand if it's too much tbh, but that was my initial thought

Reply:
That would require far too much querying
Let's keep it simple to just shop name and description.